server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexActions.scala

```diff
@@ -0,0 +1,114 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import play.api.libs.json.Writes
+import play.api.mvc.Result
+import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenResult
+import uk.gov.ons.addressIndex.model.{BritishStandard7666, PostcodeAddressFile}
+import uk.gov.ons.addressIndex.model.db.index.{NationalAddressGazetteerAddresses, PostcodeAddressFileAddresses}
+import uk.gov.ons.addressIndex.model.server.response._
+import uk.gov.ons.addressIndex.server.controllers.PlayHelperController
+import uk.gov.ons.addressIndex.model.AddressScheme._
+import scala.concurrent.{ExecutionContext, Future}
+
+trait AddressIndexActions { self: AddressIndexCannedResponse with PlayHelperController =>
```

Comment:
So why is it needed here? Do you want to test it with another implementation of `AddressIndexCannedResponse with PlayHelperController` ?